lib/Stats.js

```diff
@@ -630,20 +627,12 @@ class Stats {
 				colors.normal(prefix);
 				let sum = 0;
 				const path = [];
-				let current = module;
-				while(current.issuer) {
-					path.unshift(current = current.issuer);
+				if(module.issuer) {
```

Comment:
Looks like you discovered a bug here.

issuer is a string, but this code assumes it's a module object.

We need to move this code into the `toJSON` method where we have access to the issuer chain.

-> separate PR